sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -855,6 +855,24 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val SHUFFLE_UCX_USE_SOCKADDR = conf("spark.rapids.shuffle.ucx.useSockaddr")
```

Comment:
suggest, `SHUFFLE_UCX_LISTENER_ENABLED` => `spark.rapids.shuffle.ucx.listener.enabled`

also lets make `internal()` for now.